docs/ops/matrix/MatMul_1.md

```diff
@@ -57,14 +57,17 @@ Two attributes, `transpose_a` and `transpose_b` specify embedded transposition f
 
 **Inputs**:
 
-*   **1**: Input batch of matrices A. Rank >= 1. Required.
+*   **1**: Tensor of type *T* with matrices A. Rank >= 1. Required.
 
-*   **2**: Input batch of matrices B. Rank >= 1. Required.
+*   **2**: Tensor of type *T* with matrices B. Rank >= 1. Required.
 
 **Outputs**
 
-*   **1**: Tensor with results of the multiplication.
+*   **1**: Tensor of type *T* with results of the multiplication.
 
+**Types**:
+
+* *T*: any supported floating point or integer type.
```

Comment:
I have mixed feeling about this _supported_ word, it is very vague. I guess it means that some plugins may not support all precissions. If it is the case I see two options here, either we specify only the common subset supported by all or we add definition in some common place what _supported_ mean for each plugin and reference it across whole documentation. @lazarevevgeny what you think about it ? 

@mitruska did you check that CPU plugin supports also integer type ?

Reply:
So have mixed feelings too, 
but I was able to run MatMul single_layer tests on CPU with the following precisions:
  
        InferenceEngine::Precision::FP32,
        InferenceEngine::Precision::FP16,
        InferenceEngine::Precision::BF16,
        InferenceEngine::Precision::I32,
        InferenceEngine::Precision::I64
        
And backend test with `int32` seems also works on CPU.
I remember that some time ago there were some votes to not listed specific types in the spec, and many of the specifications use the 'supported' word but I'm not sure what is the best solution here.

Reply:
Let's keep the text as it is currently updated by @mitruska .